src/graphics/program-lib/chunks/standard/frag/sheenGloss.js

```diff
@@ -0,0 +1,27 @@
+export default /* glsl */`
+#ifdef MAPFLOAT
+uniform float material_sheenGlossiness;
+#endif
+
+#ifdef MAPTEXTURE
+uniform sampler2D texture_sheenGlossinessMap;
+#endif
+
+void getSheenGlossiness() {
+    float sheenGlossiness = 1.0;
+
+    #ifdef MAPFLOAT
+    sheenGlossiness *= material_sheenGlossiness;
+    #endif
+
+    #ifdef MAPTEXTURE
+    sheenGlossiness *= texture2D(texture_sheenGlossinessMap, $UV, textureBias).$CH;
+    #endif
+
+    #ifdef MAPVERTEX
+    sheenGlossiness *= saturate(vVertexColor.$VC);
+    #endif
+
+    sheenGlossiness += 0.0000001;
```

Comment:
```suggestion
    sheenGlossiness += 0.0000001;
    
    sGlossiness = sheenGlossiness;
```

Reply:
This chunk is calculating a local variable and not setting the global?

Reply:
Ah yes, I merged in the change but did a rebase on main and force push, which reverted it. It's fixed again.